### PR TITLE
Use f-strings everywhere

### PR DESCRIPTION
**Pull request by schneiderfelipe**
This converts all string concatenations, `%` syntax and `format` calls to use [f-strings](https://peps.python.org/pep-0498/), which is available from Python 3.6+. I used [flynt](https://github.com/ikamensh/flynt) to make the conversion automatically (`flynt -ll 1000 -tc -a .`). EDIT: this probably overlaps a bit with #67.

(As a bonus, using [f-strings can sometimes improve performance](https://github.com/cclib/cclib/pull/1121#issue-1181418010) since it's built into the syntax and fewer function calls are required. I haven't tested that here though.)

### CODE REVIEW

File: omnizart/constants/datasets.py [hunk 1]
```diff
@@ -165,7 +165,7 @@ def download(cls, save_path="./"):
         or filter out some files.
         """
         ensure_path_exists(save_path)
-        save_name = cls.__name__.replace("Structure", "") + ".zip"
+        save_name = f"{cls.__name__.replace('Structure', '')}.zip"
```
Comment (BreezeWhite) [inline omnizart/constants/datasets.py:168]: Please revert this change. The original one is more clear than the f-string one.

**Review by BreezeWhite — CHANGES_REQUESTED** [review]

File: omnizart/cli/cli.py [hunk 2]
```diff
@@ -108,43 +108,43 @@ def download_checkpoints(output_path):
...
             "save_as": "checkpoints/vocal/vocal_contour/variables/variables.data-00000-of-00001",
         },
         "beat": {
-            "fid": release_url + "beat_blstm@variables.data-00000-of-00001",
+            "fid": f"{release_url}beat_blstm@variables.data-00000-of-00001",
             "save_as": "checkpoints/beat/beat_blstm/variables/variables.data-00000-of-00001",
         },
         "patch_cnn_melody": {
-            "fid": release_url + "patch_cnn_melody@variables.data-00000-of-00001",
+            "fid": f"{release_url}patch_cnn_melody@variables.data-00000-of-00001",
```
[the remark this answers is not in the document]
Comment (BreezeWhite) [reply omnizart/cli/cli.py:147]: Notice to change all the others above this line.

File: omnizart/callbacks.py [hunk 3]
```diff
@@ -227,8 +227,7 @@ def _save_model(self, epoch, logs):
                     else:
                         if self.monitor_op(current, self.best):
                             if self.verbose > 0:
-                                print('\nEpoch %05d: %s improved from %0.5f to %0.5f, \
-                                    saving model to %s' % (epoch + 1, self.monitor, self.best, current, filepath))
+                                print(f'\nEpoch {int(epoch + 1):05}: {self.monitor} improved from {self.best:0.5f} to {current:0.5f},                                     saving model to {filepath}')
```
[the remark this answers is not in the document]
Comment (schneiderfelipe) [reply omnizart/callbacks.py]: What about now (I've split the string into two, but preserved the f-string syntax)?

File: omnizart/constants/datasets.py (hunk 1 above)
Comment (schneiderfelipe) [reply omnizart/constants/datasets.py]: Sure, agreed. Just reverted it.

File: omnizart/cli/cli.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (schneiderfelipe) [reply omnizart/cli/cli.py]: Perfect, done!

**Comment by BreezeWhite** [comment]
@schneiderfelipe Please check the CI errors and fix them, thx.
